probe/endpoint/nat.go

```diff
@@ -16,16 +16,18 @@ type endpointMapping struct {
 	rewrittenPort int
 }
 
-type natmapper struct {
-	*Conntracker
+// NATMapper rewrites a report to deal with NAT's connections
+type NATMapper struct {
+	Conntracker
 }
 
-func newNATMapper() (*natmapper, error) {
+// NewNATMapper is exposed for testing
+func NewNATMapper() (*NATMapper, error) {
```

Comment:
I think now you can have all the NATMapper methods on a value receiver, since Conntracker is an interface...